api/v1/helpers/verbs/doDelete.js

```diff
@@ -50,6 +53,8 @@ function doDelete(req, res, next, props) {
 
     // when a resource is deleted, delete all its associations too
     u.deleteAllAssociations(o, assocNames);
+    resultObj.retval = o.dataValues;
+    u.logAPI(req, resultObj);
```

Comment:
u.logAPI and logAPI were different functions. Renamed logAPI to logAuditAPI to avoid confusion.